### PR TITLE
B/add data

### PR DESCRIPTION
**Pull request by brandonhoughton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,12 +75,299 @@ on how they were trained and the exact reward schedule.
...
+
+  </details>
+
+The following is a list of the available versions:
+
+* **6.x** Core recorder features subject to change [:arrow_down: index file](todo.link)
+  * 6.9 First feature complete recorder version [:arrow_down: index file](todo.link)
+  * 6.10 Fixes mouse scaling on Mac when gui is open [:arrow_down: index file](todo.link)
+  * 6.11 Tracks the hotbar slot [:arrow_down: index file](todo.link)
+  * 6.13 Springing, swap-hands, ... (see commits below) [:arrow_down: index file](todo.link)
```
Comment (pzhokhov) [inline README.md:104]: Sprinting?

Comment (brandonhoughton) [reply README.md:104]: Lol good catch

**Review by Miffyli — APPROVED** [review]
LGTM with small comment: I do not see cursors in video clips when people open the inventory. Did you guys add these later in the training scripts? Clarification would be nice.

Otherwise good to merge :)

**Comment by brandonhoughton** [comment]
Re previous comment I am adding the disclaimer about the cursor not being present - we add it in later as a data augmentation

**Pull request merged**